packer/templates/ubuntu.json

```diff
@@ -0,0 +1,64 @@
+{
+  "variables": {
+    "ansible_roles_path": ".galaxy",
+    "base_image": null,
+    "distro": null,
+    "cloud_auth_url": null,
+    "cloud_user": null,
+    "cloud_pass": null,
+    "cloud_network": null,
+    "cloud_tenant": null,
+    "cloud_user_data": null,
+    "ssh_user": null,
+    "ssh_proxy_host": ""
+  },
+  "builders": [
+    {
+      "name": "vexxhost",
+      "image_name": "ZZCI - {{user `distro`}} - snapcraft - {{isotime \"20060102-1504\"}}",
```

Comment:
You're you're naming the built image with an identifier of snapcraft but the file is named ubuntu.json please change the file name to match the image identifier.